portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -70,11 +74,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     storeComments(name, email, comment, timestamp);
     response.sendRedirect("/index.html");
   }
-
-  private static String buildComment(Entity entity) {
+  
+  private static String buildComment(Entity entity, String prevLanguageCode, String languageCode) {
```

Comment:
Ditto. Use sourceLanguageCode and targetLanguageCode.

Reply:
Done